build.go

```diff
@@ -137,8 +139,27 @@ func buildUncached(installables []string, buildArgs []string) ([]string, error)
 	}
 
 	var builtDrvs []string
+
+	file, err := os.Create("./drvs")
+	if err != nil {
+		_ = fmt.Errorf("cannot write drvs to file: %s\n", err)
+	}
+	defer file.Close()
+
 	for drv := range missingDrvs {
+		c := exec.Command("nix", "show-derivation", "--experimental-features", "nix-command", drv)
```

Comment:
If you are already adding third-party dependencies, it would be more straight forward to parse the derivation file directly using https://github.com/nix-community/go-nix#pkgderivation

Reply:
ah yeah, didn't realize that was an option, good idea.